.circleci/config.yml

```diff
@@ -335,12 +335,37 @@ jobs:
           destination: capybara
       - *upload-coverage
 
+  docker-build:
+    resource_class: small
+    docker:
+      - image: docker:stable
+        environment:
+          DOCKER_COMPOSE: /usr/bin/docker-compose
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Login to Container Registry
+          command: |
+            docker login --username "${DOCKER_USERNAME}"\
```

Comment:
Why do we need to login? The base images are publicly available I think

Reply:
Yeah, It was because of the dependencies for the smoke test. Guess it could be removed now.